nvcf/_nvcf_deploy.sh

```diff
@@ -0,0 +1,3 @@
+ngc cloud-function function deploy create \
```

Comment:
Can we add a 1-line comment at the top of this script that describes what it is doing?